### PR TITLE
Wait for device reboot

### PR DESCRIPTION
**Pull request by vcdmp**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

Wait for device reboot

[fwupdtool_output.txt](https://github.com/fwupd/fwupd/files/15381437/fwupdtool_output.txt)

### CODE REVIEW

File: plugins/logitech-tap/fu-logitech-tap-hdmi-device.c [hunk 1]
```diff
@@ -500,6 +500,7 @@ fu_logitech_tap_hdmi_device_init(FuLogitechTapHdmiDevice *self)
 	fu_udev_device_add_flag(FU_UDEV_DEVICE(self), FU_UDEV_DEVICE_FLAG_OPEN_WRITE);
 	fu_udev_device_add_flag(FU_UDEV_DEVICE(self), FU_UDEV_DEVICE_FLAG_OPEN_NONBLOCK);
 	fu_udev_device_add_flag(FU_UDEV_DEVICE(self), FU_UDEV_DEVICE_FLAG_IOCTL_RETRY);
+	fu_device_add_internal_flag(FU_DEVICE(self), FU_DEVICE_INTERNAL_FLAG_REPLUG_MATCH_GUID);
```
Comment (hughsie) [inline plugins/logitech-tap/fu-logitech-tap-hdmi-device.c:503]: do the physical and logical IDs change during flashing?

File: plugins/logitech-tap/fu-logitech-tap-device.c [hunk 2]
```diff
@@ -30,7 +30,7 @@ fu_logitech_tap_device_init(FuLogitechTapDevice *self)
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_SIGNED_PAYLOAD);
 	fu_device_register_private_flag(FU_DEVICE(self),
 					FU_LOGITECH_TAP_HDMI_DEVICE_FLAG_NEEDS_REBOOT,
```
Comment (hughsie) [inline plugins/logitech-tap/fu-logitech-tap-device.c:32]: maybe rename this FU_LOGITECH_TAP_HDMI_DEVICE_FLAG_SENSOR_NEEDS_REBOOT

**Review by superm1 — COMMENTED** [review]
split into two commits.  Also; do you want these in 1.9.x?

**Comment by vcdmp** [comment]
> split into two commits. Also; do you want these in 1.9.x?

Split: https://github.com/fwupd/fwupd/pull/7270
Yes for 1.9.x

File: plugins/logitech-tap/fu-logitech-tap-device.c (hunk 2 above)
Comment (vcdmp) [reply plugins/logitech-tap/fu-logitech-tap-device.c:32]: Done: https://github.com/fwupd/fwupd/pull/7270

**Comment by superm1** [comment]
Can you please rebase on main and force push?  No merge commits please.

**Comment by superm1** [comment]
I see another push with a merge request, but it's not rebased, please try again.  You can do it like this:

```
git rebase -i origin/main
git push --force origin
```

**Review by superm1 — APPROVED** [review]

**Pull request merged**